vector/src/androidTest/java/im/vector/app/ui/UiTestBase.kt

```diff
@@ -29,8 +31,17 @@ import im.vector.app.R
 import im.vector.app.espresso.tools.waitUntilActivityVisible
 import im.vector.app.features.home.HomeActivity
 import im.vector.app.waitForView
+import java.lang.Thread.sleep
 
 class UiTestBase {
+
+    fun enableDeveloperMode() {
+        PreferenceManager.getDefaultSharedPreferences(InstrumentationRegistry.getInstrumentation().targetContext)
+                .edit()
```

Comment:
Nit: can't we use `edit {}` extension from tests?
By the way this is a nice way to do it, I was thinking of navigating to the settings with the UI (in allScreensTest()) to change this pref :):):)